workspace-loader/webpack.dev.js

```diff
@@ -0,0 +1,51 @@
+/*******************************************************************************
+ * Copyright (c) 2015-2018 Red Hat, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ *******************************************************************************/
+
+const merge = require('webpack-merge');
+const common = require('./webpack.common.js');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = merge(common, {
+    devtool: 'inline-source-map',
```

Comment:
sourcemap is working if not enabled in tsconfig ?

Reply:
I have just tested when disabled in tsconfig. It works but when debugging source files have a lot of extra information.